poetry.lock

```diff
@@ -679,11 +827,16 @@ version = "0.6.0"
 [package.dependencies]
 more-itertools = "*"
 
+[package.extras]
+docs = ["sphinx", "jaraco.packaging (>=3.2)", "rst.linker (>=1.9)"]
+testing = ["pathlib2", "contextlib2", "unittest2"]
+
 [extras]
 django = ["django"]
+flask = ["flask"]
```

Comment:
Hey @jaydenwindle @patrick91 Poetry does not recognise optional dependencies in `dev-dependencies` when installing with `--extras`  flag as shown above. The only workaround is moving `pytest-django` and `pytest-flask` to `dependencies.

Reply:
@johnchuks I think it is fine to have all the dev dependencies always declared :) in dev we always want to test flask and django :)